crates/symbolicator/src/services/cacher.rs

```diff
@@ -446,7 +515,7 @@ impl<T: CacheItemRequest> Cacher<T> {
         // cache_path is None when caching is disabled.
         let cache_path = self.get_cache_path(&key, T::VERSIONS.current);
         if let Some(ref path) = cache_path {
-            if let Some(item) = self.lookup_cache(&request, &key, path)? {
+            if let Some(item) = self.lookup_local_cache(&request, &key, path)? {
```

Comment:
if i understand this correctly, this means that we're actually doing the local lookup first, and then falling back to the shared cache if there isn't a hit there. 

is that what the code is doing? if so, aren't the fallbacks reversed in this case, since we want to fall back to local if shared doesn't have a hit?

Reply:
an additional tiny nit is that perhaps we would want to revisit the name of `compute_memoized` and `compute` if one of the major differences is the cache that's being used.

Reply:
> we want to fall back to local if shared doesn't have a hit

we always want to look into the local cache first, since that is the quickest. It just goes straight to sync fs APIs, and they are supposed to be very quick.
the *second layer* is going via network to a shared cache, which might take a longer time since it needs to download stuff.
only after that, we try recomputing from the source.

Reply:
looks like i had it backwards, thanks for the explanation!